packages/ui/src/design-system/control-buttons/icon-button.component.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ReactNode } from 'react';
+import React from 'react';
+
+import cs from 'classnames';
+
+import { SkeletonButton } from '../buttons';
+
+import * as cx from './control-button.css';
+import { Scheme } from './types';
+
+import type { ButtonProps } from '../buttons';
+
+type Props = Omit<ButtonProps, 'className' | 'label'> & {
+  icon?: ReactNode;
+};
```

Comment:
The common part of this type repeats 4 times. Please extract it it as a shared type and reuse
```suggestion
import { ControllButtonCommonProps } from './types';
type IconProps = Omit<ControllButtonCommonProps, 'label'>;
```

Reply:
Resolved